src/lib/redirects-manager-context.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import { v4 } from 'uuid';
+import { saveState, checkRepeatedRequests } from './utils';
+
+export interface RedirectsManagerContextInterface {
+    store: RedirectionsStore;
+    wildcard: boolean;
+    saving?: boolean;
+    lastSave?: Date;
+    lastModification?: Date;
+    filterBy?: string;
+    perPage?: number;
+    currentPage?: number;
+    imported?: boolean;
+}
+
+export interface RedirectionProps {
+    id: string;
+    request?: string;
+    destination?: string;
+    modificationDate?: any;
+    warningRequestDuplication?: boolean;
+}
+
+export interface RedirectionsStore extends Array<RedirectionProps> { }
```

Comment:
`type RedirectionsStore = RedirectionProps[]` would be preferred - you can also use `RedirectionProps[]` without a separate type

Reply:
applied